Docs/Configuration.tex

```diff
@@ -1132,27 +1131,26 @@ \subsection{Quirks Properties}\label{acpipropsquirks}
   \textbf{Description}: Attempt to heuristically relocate ACPI memory
   regions. Not recommended.
 
-  ACPI tables are often generated dynamically by underlying firmware
+  ACPI tables are often generated dynamically by the underlying firmware
   implementation. Among the position-independent code, ACPI tables may
-  contain physical addresses of MMIO areas used for device
-  configuration, usually grouped in regions (e.g.
+  contain the physical addresses of MMIO areas used for device
+  configuration, usually grouped by region (e.g.
   \texttt{OperationRegion}). Changing firmware settings or hardware
   configuration, upgrading or patching the firmware inevitably leads to
-  changes in dynamically generated ACPI code, which sometimes lead to
-  the shift of the addresses in aforementioned \texttt{OperationRegion}
+  changes in dynamically generated ACPI code, which sometimes results in
+  the shift of the addresses in the aforementioned \texttt{OperationRegion}
   constructions.
 
-  For this reason it is very dangerous to apply any kind of
-  modifications to ACPI tables. The most reasonable approach is to make
-  as few as possible changes to ACPI and try to not replace any tables,
-  especially DSDT. When this is not possible, then at least attempt to
-  ensure that custom DSDT is based on the most recent DSDT or remove
-  writes and reads for the affected areas.
+  For this reason, the application of modifications to ACPI tables is extremely
+  risky. The best approach is to make as few as possible changes to ACPI tables
+  and to avoid replacing any tables, particularly DSDT tables. When this cannot be
+  avoided, ensure that any custom DSDT tables are based on the most recent DSDT
+  tables or attempt to remove reads and writes for the affected areas.
 
-  When nothing else helps this option could be tried to avoid stalls at
-  \texttt{PCI\ Configuration\ Begin} phase of macOS booting by
-  attempting to fix the ACPI addresses. It does not do magic, and only
-  works with most common cases. Do not use unless absolutely required.
+  When nothing else helps, this option could be tried to avoid stalls at
+  \texttt{PCI\ Configuration\ Begin} phase of macOS booting by attempting to fix
+  the ACPI addresses. It is not a magic bullet however, and only works with the
+  most typical cases. Do not use unless absolutely required.
```

Comment:
Add "as it may cause an opposite effect on some platforms resulting in boot failures".